selectinf/randomized/query.py

```diff
@@ -979,6 +983,10 @@ def selective_MLE(self,
             Arguments passed to solver.
         """
 
+        self.M1 = self.M1 * dispersion
```

Comment:
General comment -- I think of `dispersion` as $\sigma^2$ not $\sigma$. How is it coded here?